### PR TITLE
Filament v3 upgrade

### PR DESCRIPTION
**Pull request by ramsesz**
Starting point for filament v3 support

Credits to @dododedodonl because I used his [changes](https://github.com/DutchCodingCompany/filament-socialite/pull/46) as a starting point.

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-Route::domain(config('filament.domain'))
-    ->middleware(config('filament.middleware.base'))
+use DutchCodingCompany\FilamentSocialite\Http\Controllers\SocialiteLoginController;
+use Filament\Facades\Filament;
+use Illuminate\Support\Facades\Route;
+
+$panel = Filament::getDefaultPanel();
+
+Route::domain($panel->getDomain())
+    ->middleware($panel->getMiddleWare())
```
Comment (ramsesz) [inline routes/web.php:10]: This was the most important update to make it work

Comment (dododedodonl) [reply routes/web.php:10]: Thanks for pointing it out ;)

**Comment by dododedodonl** [comment]
Thank you for your work. I'll merge it to the other branch; and continue to work on it to incorporate multi panel support.

**Comment by KarelBrijs** [comment]
Hi @dododedodonl 
Just wanted to check in if there is any update on the Filament V3 status? I know this is an open-source effort so I also wanted to check in if there is anything to be done by the community to accelerate this change? I see in the Filament Discord there are more people asking for an update on V3 status of this plugin. Anyhow already many thanks for your efforts! 👍

**Comment by bramr94** [comment]
@KarelBrijs I've got some time planned to work on this, maybe tomorrow, otherwise hopefully next week.

**Comment by KarelBrijs** [comment]
@bramr94 that would be amazing! If there's anything I can do, let me know!

**Comment by bramr94** [comment]
@ramsesz @KarelBrijs I'm going to close this PR. I've continued @dododedodonl work on this PR https://github.com/DutchCodingCompany/filament-socialite/pull/46. 

I've got some issues with showing error messages on the form after a redirect, maybe one of you got an idea how to fix it? If you want to test the PR in its current state would be great as well.

**Comment by KarelBrijs** [comment]
@bramr94 Thanks alot for your work, I'll check it out tonight or tomorrow!

**Pull request closed without merging**